editor/js/Sidebar.Material.js

```diff
@@ -500,7 +500,7 @@ function SidebarMaterial( editor ) {
 
 				}
 
-				editor.execute( new SetMaterialCommand( editor, currentObject, material, currentMaterialSlot ), 'New Material: ' + materialClass.getValue() );
+				editor.execute( new SetMaterialCommand( editor, currentObject, material, currentMaterialSlot ), strings.getKey( 'command/SetMaterial' ) + ':' + materialClass.getValue() );
```

Comment:
For readability, it would be good to retain the space after the `:` sign.

Reply:
`xxx : yyy` or `xxx: yyy` ?

Reply:
`xxx: yyy` as it was before.

Reply:
ok.